lib/filter.c

```diff
@@ -146,6 +156,29 @@ static const char *filter_type_str(struct filter *filter)
 	}
 }
 
+/*
+ * mac filter match
+ * n is of type struct prefix_eth
+ * p can be of type struct ethaddr
+ */
+static int mac_filter_match(struct prefix *n, struct ethaddr *p)
```

Comment:
This entire function should now be redundant as `prefix_match` should work for Ethernet